lib/Doctrine/Compiler.php

```diff
@@ -127,7 +127,7 @@ public static function compile($target = null, $includedDrivers = array())
         $stripped = php_strip_whitespace($target);
         $fp = @fopen($target, 'w');
         if ($fp === false) {
-            throw new Doctrine_Compiler_Exception("Couldn't write compiled data. Failed to open $file");
+            throw new Doctrine_Compiler_Exception("Couldn't write compiled data. Failed to open $target");
```

Comment:
`$file` is last set in a `foreach` loop above (so it may not get set), but it seems that this error message was meant to use `$target`, just like the error message a few lines up (line 121).